src/Compilers/VisualBasic/Portable/Binding/Binder_Statements.vb

```diff
@@ -1066,9 +1066,10 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
 
             Dim symbol As LocalSymbol = GetLocalForDeclaration(name.Identifier)
 
-            Dim valueExpression As BoundExpression = Nothing
+            Dim declarationInitializer As BoundExpression = Nothing
             Dim declType As TypeSymbol = Nothing
             Dim boundArrayBounds As ImmutableArray(Of BoundExpression) = Nothing
+            Dim identifierInitializer As BoundArrayCreation = Nothing
```

Comment:
Consider moving `identifierInitializer` declaration closer to where it used, say line 1197.

Reply:
Done.

---
In reply to: [148030222](https://github.com/dotnet/roslyn/pull/22842#discussion_r148030222) [](ancestors = 148030222)